src/Backoffice/Airlines/App/Controllers/StoreAirlineController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airlines\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Airlines\App\Requests\StoreAirlineRequest;
+use Lightit\Backoffice\Airlines\App\Transformers\AirlineTransformer;
+use Lightit\Backoffice\Airlines\Domain\Actions\StoreAirlineAction;
+
+class StoreAirlineController
+{
+    public function __invoke(StoreAirlineRequest $request, StoreAirlineAction $action): JsonResponse
+    {
+        $airline = $action->execute($request->toDto());
+
+        return responder()
+            ->success($airline, AirlineTransformer::class)
+            ->respond(
+                $airline->wasRecentlyCreated
+                    ? JsonResponse::HTTP_CREATED
+                    : JsonResponse::HTTP_OK
```

Comment:
Why ? All Airlines should be recently created 
```suggestion
         JsonResponse::HTTP_CREATED
```